docs: add 'Help Needed' page (#1405)
commercetools/merchant-center-application-kit: .github/ISSUE_TEMPLATE/bug_report.md

---
name: Bug report
about: Something is not working?
title: ''
labels:
- "🐛 Type: Bug"
assignees: ''
---

**Describe the issue**

> <!-- A clear and concise description of what the issue is. -->

Reviewer: ```suggestion
<!-- A clear and concise description of what's not working. -->
```

To avoid repeating "issue"
